pkg/charts/linkerd2/values.go

```diff
@@ -47,6 +47,7 @@ type (
 		ProfileValidator              *ProfileValidator              `json:"profileValidator"`
 		Tap                           *Tap                           `json:"tap"`
 		NodeSelector                  map[string]string              `json:"nodeSelector"`
+		Tolerations                   map[string]string              `json:"tolerations"`
```

Comment:
Unlike `NodeSelector` which is a map, the `Tolerations` property is an array, so this will fail parsing the `values.yaml` file. You can instead use the `[]interface{}` to acomodate the array.

Reply:
Oh, yes
you are correct, and updated on [1abb6d7](https://github.com/linkerd/linkerd2/pull/4626/commits/1abb6d79723da4ac5b0740d0c93bc8ebc7551a34)